sw/device/lib/testing/test_framework/ottf_main.c

```diff
@@ -80,6 +81,7 @@ void _ottf_main(void) {
   // Initialize the UART to enable logging for non-DV simulation platforms.
   if (kDeviceType != kDeviceSimDV) {
     init_uart();
+    LOG_INFO("Starting test %s", kTestConfig.file);
```

Comment:
I think, This can happen everytime we reboot within a test.

Reply:
Good point, but I think it's still accurate :) It's starting the test (again).

Reply:
> Good point, but I think it's still accurate :) It's starting the test (again).

FYI, that's not true for tests that are specifically testing self-reset events. This unconditional statement may make things confusing. Log output for when tests end should be okay, though.

Reply:
Thanks @drewmacrae and @a-will, updated this line to "Running ..." as we discussed offline.